application/repositories.go

```diff
@@ -49,6 +50,9 @@ type SearchRepository interface {
 // IdentityRepository encapsulates identity
 type IdentityRepository interface {
```

Comment:
The account module should own it's own Interface, move it back to account package.